src/Test/Hspec/Formatters/Codewars.hs

```diff
@@ -1,38 +1,36 @@
 {-# LANGUAGE OverloadedStrings #-}
-module Test.Hspec.Formatters.Codewars
-  (
-    codewars
-  ) where
 
-import Data.Text (pack, unpack, replace)
+module Test.Hspec.Formatters.Codewars (codewars) where
 
-import Test.Hspec.Formatters (Formatter (..),
-                              FailureReason (..),
-                              formatException,
-                              silent,
-                              writeLine)
+import Data.Text (pack, replace, unpack)
 
-codewars :: Formatter
-codewars = silent {
-  exampleGroupStarted = \_ name -> do
-    writeLine ""
-    writeLine $ escapeLF $ "<DESCRIBE::>" ++ name
-
-, exampleGroupDone = writeLine "\n<COMPLETEDIN::>"
+import Test.Hspec.Core.Formatters.V1 (
+  FailureReason (..),
+  Formatter (..),
+  formatException,
+  silent,
+  writeLine,
+ )
 
-, exampleSucceeded = \(_, name) _ -> do
-    writeLine ""
-    writeLine $ escapeLF $ "<IT::>" ++ name
-    writeLine "\n<PASSED::>Test Passed"
-    writeLine "\n<COMPLETEDIN::>"
-
-, exampleFailed = \(_, name) _ reason -> do
-    writeLine ""
-    writeLine $ escapeLF $ "<IT::>" ++ name
-    writeLine ""
-    writeLine $ escapeLF $ reasonAsString reason
-    writeLine "\n<COMPLETEDIN::>"
-}
+codewars :: Formatter
+codewars =
+  silent
+    { exampleGroupStarted = \_ name -> do
+        writeLine ""
+        writeLine $ escapeLF $ "<DESCRIBE::>" ++ name
+    , exampleGroupDone = writeLine "\n<COMPLETEDIN::>"
+    , exampleSucceeded = \(_, name) _ -> do
+        writeLine ""
```

Comment:
:question: Are all newlines necessary? The output shows empty lines between each output.

Reply:
Yes, we're prepending a newline to prevent the solution's outputs from interfering.

Reply:
Okay, I was just wondering since here (Haskell) are empty lines between each line, and in PureScript there isn't.

Reply:
Update: After recent changes PureScript also has an empty line between each output.